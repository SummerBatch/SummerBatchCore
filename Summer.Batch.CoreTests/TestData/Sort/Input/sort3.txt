000000004800000011600000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000002076867{000000075000001404170000000000000480000000020140417 000000000000000000000000000000000000000000000000000000
000000004700000011600000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000003845780G000000008000001404170000000000000470000000020140417 000000000000000000000000000000000000000000000000000000
000000004700000011600000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000011461I000000014000001404170000000000000470000000020140417 000000000000000000000000000000000000000000000000000000
000000004700000011600000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000022923H000000037000001404170000000000000470000000020140417 000000000000000000000000000000000000000000000000000000
000000004700000011600000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000022923H000000075000001404170000000000000470000000020140417 000000000000000000000000000000000000000000000000000000
000000004700000011600000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000034385G000000075300001404170000000000000470000000020140417 000000000000000000000000000000000000000000000000000000
000000004700000011600000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000022923H000000094000001404170000000000000470000000020140417 000000000000000000000000000000000000000000000000000000
000000004700000011600000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000004465{000000003000001404170000000000000470000000020140417 000000000000000000000000000000000000000000000000000000
000000004700000011600000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000008550{000000003000001404170000000000000470000000020140417 000000000000000000000000000000000000000000000000000000
000000004700000011600000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000004370{000000003000001404170000000000000470000000020140417 000000000000000000000000000000000000000000000000000000
000000004700000011600000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000033630{000000049000001404170000000000000470000000020140417 000000000000000000000000000000000000000000000000000000
000000004700000011600000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000002660{000000055000001404170000000000000470000000020140417 000000000000000000000000000000000000000000000000000000
000000004700000011600000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000014915{000000075000001404170000000000000470000000020140417 000000000000000000000000000000000000000000000000000000
000000004700000011600000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000001339B000000028000001404170000000000000470000000020140417 000000000000000000000000000000000000000000000000000000
000000004700000011600000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000002008H000000028000001404170000000000000470000000020140417 000000000000000000000000000000000000000000000000000000
000000004700000011600000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000016126B000000055000001404170000000000000470000000020140417 000000000000000000000000000000000000000000000000000000
000000004700000011600000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000024189C000000055000001404170000000000000470000000020140417 000000000000000000000000000000000000000000000000000000
000000004700000011600000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000003013B000000059000001404170000000000000470000000020140417 000000000000000000000000000000000000000000000000000000
000000004700000011600000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000004519H000000059000001404170000000000000470000000020140417 000000000000000000000000000000000000000000000000000000
000000004700000011600000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000423G000000042000001404170000000000000470000000020140417 000000000000000000000000000000000000000000000000000000
000000004700000011600000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000792A000000042000001404170000000000000470000000020140417 000000000000000000000000000000000000000000000000000000
000000004700000011600000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000002C000000002000001404170000000000000470000000020140417 000000000000000000000000000000000000000000000000000000
000000004700000011600000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000002C000000002000001404170000000000000470000000020140417 000000000000000000000000000000000000000000000000000000
000000004700000011600000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000020I000000002000001404170000000000000470000000020140417 000000000000000000000000000000000000000000000000000000
000000004700000011600000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000022{000000002000001404170000000000000470000000020140417 000000000000000000000000000000000000000000000000000000
000000004700000011600000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000022{000000002000001404170000000000000470000000020140417 000000000000000000000000000000000000000000000000000000
000000004700000011600000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000197G000000002000001404170000000000000470000000020140417 000000000000000000000000000000000000000000000000000000
000000004700000011600000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000011I000000002000001404170000000000000470000000020140417 000000000000000000000000000000000000000000000000000000
000000004700000011600000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000011I000000002000001404170000000000000470000000020140417 000000000000000000000000000000000000000000000000000000
000000004700000011600000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000106G000000002000001404170000000000000470000000020140417 000000000000000000000000000000000000000000000000000000
000000004700000011600000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000041B000000003000001404170000000000000470000000020140417 000000000000000000000000000000000000000000000000000000
000000004700000011600000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000041B000000003000001404170000000000000470000000020140417 000000000000000000000000000000000000000000000000000000
000000004700000011600000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000370D000000003000001404170000000000000470000000020140417 000000000000000000000000000000000000000000000000000000
000000004700000011600000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000024B000000029000001404170000000000000470000000020140417 000000000000000000000000000000000000000000000000000000
000000004700000011600000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000024B000000029000001404170000000000000470000000020140417 000000000000000000000000000000000000000000000000000000
000000004700000011600000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000217H000000029000001404170000000000000470000000020140417 000000000000000000000000000000000000000000000000000000
000000004700000011600000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000006I000000043000001404170000000000000470000000020140417 000000000000000000000000000000000000000000000000000000
000000004700000011600000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000006I000000043000001404170000000000000470000000020140417 000000000000000000000000000000000000000000000000000000
000000004700000011600000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000062{000000043000001404170000000000000470000000020140417 000000000000000000000000000000000000000000000000000000
000000004700000011600000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000114A000000075000001404170000000000000470000000020140417 000000000000000000000000000000000000000000000000000000
000000004700000011600000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000114A000000075000001404170000000000000470000000020140417 000000000000000000000000000000000000000000000000000000
000000004700000011600000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000001027A000000075000001404170000000000000470000000020140417 000000000000000000000000000000000000000000000000000000
000000004700000011600000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000067G000000075300001404170000000000000470000000020140417 000000000000000000000000000000000000000000000000000000
000000004700000011600000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000067G000000075300001404170000000000000470000000020140417 000000000000000000000000000000000000000000000000000000
000000004700000011600000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000609C000000075300001404170000000000000470000000020140417 000000000000000000000000000000000000000000000000000000
