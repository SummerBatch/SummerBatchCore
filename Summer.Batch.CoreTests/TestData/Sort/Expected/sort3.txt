000000004700000011600000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000004084594E000000002000001404170000000000000470000000020140417 000000000000000000000000000000000000000000000000000000
000000004800000011600000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000002076867{000000075000001404170000000000000480000000020140417 000000000000000000000000000000000000000000000000000000
